.github/workflows/release.yaml

```diff
@@ -163,7 +165,30 @@ jobs:
         id: get-version
         run: |
           ./radius/.github/scripts/release-get-version.sh ${{ steps.get-supported-versions.outputs.result }} radius
+      - name: Check if release branch exists
+        id: release-branch-exists
+        uses: actions/github-script@v6
+        with:
+          result-encoding: string
+          script: |
+            const { data } = await github.rest.repos.getBranch({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              branch: '${{ steps.get-version.outputs.release-branch-name }}',
+            })
+
+            if (data.name == '${{ steps.get-version.outputs.release-branch-name }}') {
+              return 'true'
+            } else {
+              return 'false'
+            }
+      - name: Stop workflow if release branch exists and the event is a push to main
+        if: steps.release-branch-exists == 'true' && github.ref == 'refs/heads/main' && github.event_name == 'push'
+        run: |
+          echo "Release branch ${{ steps.get-version.outputs.release-branch-name }} already exists. Stop the workflow."
+          echo "STOPPED=true" >> $GITHUB_ENV
```

Comment:
nit: consider renaming variable to something more descriptive IS_WORKFLOW_STOPPED / WORKFLOW_STOPPED